features/src/main/feature/camel-features.xml

```diff
@@ -1617,42 +1617,36 @@
-<!--        <bundle dependency='true'>mvn:org.apache.httpcomponents/httpclient-osgi/${httpclient4-version}</bundle>-->
-<!--        <bundle dependency='true'>mvn:org.apache.httpcomponents/httpasyncclient-osgi/${httpasyncclient-version}</bundle>-->
-<!--        <bundle dependency='true'>mvn:com.fasterxml.jackson.core/jackson-core/${jackson2-version}</bundle>-->
-<!--        <bundle dependency='true'>mvn:com.fasterxml.jackson.core/jackson-databind/${jackson2-version}</bundle>-->
-<!--        <bundle dependency='true'>mvn:com.fasterxml.jackson.core/jackson-annotations/${jackson2-version}</bundle>-->
-<!--        <bundle dependency='true'>mvn:com.fasterxml.jackson.dataformat/jackson-dataformat-xml/${jackson2-version}</bundle>-->
-<!--        <bundle dependency='true'>mvn:com.fasterxml.jackson.module/jackson-module-jaxb-annotations/${jackson2-version}</bundle>-->
-<!--        <bundle dependency='true'>mvn:org.codehaus.woodstox/stax2-api/${stax2v4-api-bundle-version}</bundle>-->
-<!--        <bundle dependency='true'>mvn:com.fasterxml/aalto-xml/0.9.10</bundle>-->
-<!--        <bundle dependency='true'>mvn:io.netty/netty-common/${netty-version}</bundle>-->
-<!--        <bundle dependency='true'>mvn:io.netty/netty-transport/${netty-version}</bundle>-->
-<!--        <bundle dependency='true'>mvn:io.netty/netty-transport-native-unix-common/${netty-version}</bundle>-->
-<!--        <bundle dependency='true'>mvn:io.netty/netty-buffer/${netty-version}</bundle>-->
-<!--        <bundle dependency='true'>mvn:io.netty/netty-handler/${netty-version}</bundle>-->
-<!--        <bundle dependency='true'>mvn:io.netty/netty-resolver/${netty-version}</bundle>-->
-<!--        <bundle dependency='true'>mvn:io.netty/netty-codec/${netty-version}</bundle>-->
-<!--        <bundle dependency='true'>mvn:io.netty/netty-codec-http/${netty-version}</bundle>-->
-<!--        <bundle dependency='true'>mvn:io.netty/netty-tcnative-classes/${netty-tcnative-classes-version}</bundle>-->
-<!--        <bundle dependency='true'>mvn:org.apache.camel/camel-olingo4-api/${project.version}</bundle>-->
-<!--        <bundle>mvn:org.apache.camel/camel-olingo4/${project.version}</bundle>-->
-<!--    </feature>-->
+    <feature name='camel-olingo4' version='${project.version}' start-level='50'>
+        <feature version='${project.version}'>camel-core</feature>
+        <feature version="[4,5)">http-client</feature>
+        <feature>jakarta-servlet</feature>
+        <feature>jackson</feature>
+        <bundle dependency='true'>mvn:org.apache.olingo/odata-commons-api/5.0.0</bundle>
+        <bundle dependency='true'>mvn:org.apache.olingo/odata-commons-core/5.0.0</bundle>
+        <bundle dependency='true'>mvn:org.apache.olingo/odata-client-api/5.0.0</bundle>
+<!--&lt;!&ndash;        &lt;!&ndash;TODO remove wrap protocol when https://issues.apache.org/jira/browse/OLINGO-1206 is resolved&ndash;&gt;&ndash;&gt;-->
```

Comment:
Let's fix this comment or remove it.
It should be `<!-- TODO remove wrap protocol when https://issues.apache.org/jira/browse/OLINGO-1206 is resolved -->`

Reply:
fixed, thans